x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -304,13 +305,39 @@ const getSavedObjectTypes = (
             type: { type: 'keyword' },
           },
         },
+        package_assets: {
+          type: 'nested',
+          properties: {
+            id: { type: 'keyword' },
+            type: { type: 'keyword' },
+          },
+        },
         install_started_at: { type: 'date' },
         install_version: { type: 'keyword' },
         install_status: { type: 'keyword' },
         install_source: { type: 'keyword' },
       },
     },
   },
+  [ASSETS_SAVED_OBJECT_TYPE]: {
```

Comment:
Do you need to add this new type to `allSavedObjectTypes` so that your users are granted access to these new saved objects? It's probably not strictly necessary since you still require `superuser` privileges for Fleet, and superusers can do everything regardless of what you specify here

https://github.com/elastic/kibana/blob/aa07f5c1d0868a7103e0dd5730776b205b6a868a/x-pack%2Fplugins%2Ffleet%2Fserver%2Fplugin.ts#L104-L112